core/src/stages/generate_grasp_pose.cpp

```diff
@@ -54,7 +54,7 @@ GenerateGraspPose::GenerateGraspPose(const std::string& name) : GeneratePose(nam
 	p.declare<std::string>("object");
 	p.declare<double>("angle_delta", 0.1, "angular steps (rad)");
 
-	p.declare<boost::any>("pregrasp", "pregrasp posture");
+	p.declare<boost::any>("pregrasp", std::string(""), "pregrasp posture");
```

Comment:
Out of curiosity, is there a reason to use `boost::any` rather than `std::string` directly .? despite the only way we use the `pregrasp` property is as `std::string`

Reply:
Very good point. I think, the idea was to enable some message alternatively. But you are right, currently, we only use strings.

Reply:
see comment below